### ISSUE TITLE
Pagination active page link contrast

### ISSUE THREAD

**Issue by ppvg** [minvws/nl-rdo-manon#428]
From independent accessibility report, SC 1.4.11:

<blockquote lang="nl">
Op pagina <a href="https://minvws.github.io/nl-rdo-manon/documentation/components/pagination.html">components/pagination.html</a> wordt met een geel vierkant de actieve pagina aangegeven. Dit element heeft nu niet genoeg contrast. Het contrast is 1,7:1, dit moet minstens 3,0:1 zijn. Het is hier ook mogelijk om ook op een andere manier aan te geven dat het onderdeel actief is, bijvoorbeeld door de tekst vet te drukken. Dit is zo ook bij sommige andere componenten, bijvoorbeeld op pagina <a href="https://minvws.github.io/nl-rdo-manon/documentation/components/tabs.html">components/tabs.html</a>, gedaan.
</blockquote>

The documentation theme (`soft`) is using a background color for the pagination's active page link which provides insufficient contrast.

**Comment by reinschaap** [comment]
test geslaagd
